src/main/resources/schema.sql

```diff
@@ -0,0 +1,47 @@
+
+CREATE DATABASE IF NOT EXISTS airlines;
+CREATE USER 'user'@'localhost' IDENTIFIED BY 'user';
+GRANT ALL PRIVILEGES ON airlines.* TO 'user'@'localhost';CREATE DATABASE IF NOT EXISTS airlines;
+CREATE USER 'user'@'localhost' IDENTIFIED BY 'user';
+GRANT ALL PRIVILEGES ON airlines.* TO 'user'@'localhost';
+
+USE airlines;
+
+DROP TABLE IF EXISTS `airplanes`;
```

Comment:
` sign could be omitted, it is not necessary:
```
CREATE TABLE crews (
                         id INT AUTO_INCREMENT PRIMARY KEY,
                         name VARCHAR(50) NOT NULL
);
...
CONSTRAINT FK_airplanes_crews FOREIGN KEY (crews_id) REFERENCES crews(id)
```

Reply:
fixed in